locales/en-US/app.ftl

```diff
@@ -78,6 +78,10 @@ CallNodeContextMenu--transform-focus-subtree = Focus on subtree only
         specific part of the call tree. It pulls out a branch of the call tree,
         however it only does it for that single call node. All other calls
         of the function are ignored.
+CallNodeContextMenu--transform-focus-category = Focus on <strong>{ $categoryName }</strong>
```

Comment:
can you please add the word "category"?
`Focus on category <strong>{ $categoryName }</strong>`

Also can you please also add a comment with the same format as other variables? The goal is that the localizers know what this variable is.